### ISSUE TITLE
How can I configure default sort in lcov reports?

### ISSUE THREAD

**Issue by doublethefish** [istanbuljs/nyc#1504]
## Link to bug demonstration repository

n/a

## Expected Behavior

To be able to configure lcov's HTML report to sort by coverage % by default, percolating least covered files to the top.

I have not yet been able to work out how to change the default sort order generated by lcov when using the NYC middlewhere.

We have a project that mandates 100% coverage so being able to quickly find the uncovered files would be awesome.

![image](https://user-images.githubusercontent.com/2162172/210969094-e20ad393-fc39-43a2-88e9-2f1962dd8a7e.png)

## Observed Behavior

Reports are sorted by file-first.

![image](https://user-images.githubusercontent.com/2162172/210968939-3cb45207-a696-4cef-ac58-3842eb64cf13.png)

### Troubleshooting steps
- [x] still occurring when I put `cache: false` in my nyc config

## Environment Information

```
  Binaries:
    Node: 18.7.0 - /opt/homebrew/bin/node
    Yarn: 1.22.17 - /opt/homebrew/bin/yarn
    npm: 8.15.0 - /opt/homebrew/bin/npm
  npmPackages:
    nyc: ^15.1.0 => 15.1.0
    source-map-support: ^0.5.21 => 0.5.21
    ts-node: ^10.9.1 => 10.9.1
    typescript: ^4.9.4 => 4.9.4

```

**Comment by artbindu** [comment]
Yes it's good to add some configuration in `.nycrc.json` for this default sort of test report.
